pkg/cmd/api/api.go

```diff
@@ -387,11 +387,15 @@ func processResponse(resp *http.Response, opts *ApiOptions, bodyWriter, headersW
 
 	if opts.FilterOutput != "" && serverError == "" {
 		// TODO: reuse parsed query across pagination invocations
-		err = jq.Evaluate(responseBody, bodyWriter, opts.FilterOutput)
+		var jqOutput = &bytes.Buffer{}
+		err = jq.EvaluateFormatted(responseBody, jqOutput, opts.FilterOutput, opts.IO.IsStdoutTTY())
 		if err != nil {
 			return
 		}
-	} else if opts.Template != "" && serverError == "" {
+		responseBody = jqOutput
```

Comment:
I would advise against writing to a buffer and instead reverting to write straight to the `responseBody` stream. The functional change in this PR that does `responseBody = jqOutput` (and the accompanying change to the `if...else` statement chain is in my view unnecessary and potentially risky.

Reply:
With go-gh/pkg/jq now handling colorization, this issue is now moot; the code is now almost unchanged apart from the method name change, and passing in extra indent and colorize arguments.